src/ripple/rpc/handlers/AccountInfo.cpp

```diff
@@ -88,6 +105,69 @@ Json::Value doAccountInfo (RPC::Context& context)
             if (sleSigners)
                 jvSignerList.append (sleSigners->getJson (0));
         }
+        // Return queue info if that is requested
+        if (queue)
+        {
+            auto& jvQueueData = result[jss::queue_data];
+            jvQueueData = Json::objectValue;
+
```

Comment:
Nit: I think we should deprecate this pattern and instead use this
```
auto jvQueueData = Json::objectValue;
...
result[jss::queue_data] = std:::move(jvQueueData);
```
Moving an array or object is cheap. This avoids the risk that some change to `result` will invalidate the reference.